src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs

```diff
@@ -12048,16 +12080,128 @@ .locals init (MyManagedStruct V_0)
   IL_0023:  ldflda     ""MyManagedStruct.Nested.Nested1 MyManagedStruct.Nested.n""
   IL_0028:  ldc.i4     0x1c8
   IL_002d:  call       ""void MyManagedStruct.Nested.Nested1.mutate(int)""
-  IL_0032:  ldflda     ""MyManagedStruct cls1.y""
-  IL_0037:  ldflda     ""MyManagedStruct.Nested MyManagedStruct.n""
-  IL_003c:  ldflda     ""MyManagedStruct.Nested.Nested1 MyManagedStruct.Nested.n""
+  IL_0032:  ldfld      ""MyManagedStruct cls1.y""
+  IL_0037:  ldfld      ""MyManagedStruct.Nested MyManagedStruct.n""
+  IL_003c:  ldfld      ""MyManagedStruct.Nested.Nested1 MyManagedStruct.Nested.n""
   IL_0041:  ldfld      ""int MyManagedStruct.Nested.Nested1.num""
   IL_0046:  call       ""void System.Console.WriteLine(int)""
   IL_004b:  ret
 }
 ");
         }
 
```

Comment:
Let's add a test trait for PEVerifyCompat mode and mark the new tests as such. That will make it easy to remove the feature later on if we make that decision.